src/Components/profile/MissionCard.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function MissionCard() {
+  const { missions } = useSelector((store) => store.mission);
+  const missionReserved = missions.filter((data) => data.mission === true);
+  let missionCounter = 0;
+
+  missions.map((mission) => {
+    if (!mission.mission) {
+      missionCounter += 1;
+    }
+    return missionCounter;
```

Comment:
- [ ] You are almost there! I would suggest using `filter()` instead of `map()` too to render the reserved missions.

Reply:
THANK MAN LOOKS LIKE YOU APPROVED IT BUT I JUST MADE THE REQUESTED CHANGES AND THE APP DIDNT BREAK SO I WILL TRY TO USE FILTER IN THOSE TYPE OF SITUATION THNK ALOT AGAIN